pkg/cmd/scan.go

```diff
@@ -256,9 +256,21 @@ func scanRun(opts *pkg.ScanOptions) error {
 	analysis.ProviderVersion = resourceSchemaRepository.ProviderVersion.String()
 	analysis.ProviderName = resourceSchemaRepository.ProviderName
 
-	err = selectedOutput.Write(analysis)
-	if err != nil {
-		return err
+	validOutput := false
+	for _, o := range opts.Output {
+		if err = output.GetOutput(o, opts.Quiet).Write(analysis); err != nil {
+			logrus.Errorf("Computing output %s://%s failed: %v", o.Key, o.Options["path"], err.Error())
```

Comment:
We are not really computing something here, `Error writing to output '%s'` may be more appropriate here.

The responsibility to print `%s://%s` could be a func of `output.OutputConfig` struct.

```go
type OutputConfig struct {
    Key     string
    Options map[string]string
}

func (o OutputConfig) String() string {
    return fmt.Sprintf("%s://%s", o.Key, o.Options["path"])
}
```

Reply:
BTW I'm wondering why do we have a `Options map[string]string` since we only use a `path` key, maybe we should refacto this ? (in another PR of course)

Reply:
I agree, a map is a lack of type safety.